pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -54,6 +54,19 @@ type HyperConvergedSpec struct {
 	// +optional
 	CertConfig *HyperConvergedCertConfig `json:"certConfig,omitempty"`
 
+	// ResourceRequirements describes the resource requirements for the operand workloads.
+	// +optional
+	ResourceRequirements *OperandResourceRequirements `json:"resourceRequirements,omitempty"`
+
+	// Override the storage class to used for scratch space during transfer operations. The scratch space storage class
```

Comment:
to use

Reply:
This is copy&paste from CDI...

maybe "to be used"?

Reply:
"Override the storage class used for scratch space during transfer operations."